src/Application/Actions/Donations/Create.php

```diff
@@ -140,6 +142,19 @@ protected function action(Request $request, Response $response, array $args): Re
                     ),
                 ),
             );
+        } catch (ORMException | DBALServerException $ex) {
```

Comment:
don't we need to make an error log / alarm if we catch these exceptions?

Reply:
There should be two alarms so far:
1. the new "... Giving up ..." log in `DonationService`
2. the return status 500 alarm

It won't have a full trace but I think maybe for this type of exception the message is enough for it to be quite clear. Open to suggestions on tweaking it though.

Reply:
ah ok, I didn't realise returning 500 automatically generated an alarm. That should be fine then. I don't quite like relying on an alarm having been logged earlier in a different class when we catch an exception.